src/context/auth-context.js

```diff
@@ -21,7 +23,24 @@ const useProvideAuth=()=>{
                 console.log(error);
```

Comment:
I guess here you have forgotten to write the catch statement.

Reply:
i've logged the error , later on i'll add a seperate routes for internal server error

Reply:
Okay.